### ISSUE TITLE
Undefined method 'expires' for #<String:0x0...>

### ISSUE THREAD

**Issue by sqlninja** [andyjeffries/flexirest#9]
I have been swapping out Active Rest Client for Flexirest on a branch. Thus far I have swapped out the initializer and the base model inherited class to be Flexirest::Base as well as swapped out all exception handlers.

But any model I call yields this error:
```
$ s = Series.all
NoMethodError: undefined method `expires' for #<String:0x007f83be516000>
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/flexirest-1.2.11/lib/flexirest/request.rb:158:in `block in call'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.4/lib/active_support/notifications.rb:164:in `block in instrument'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.4/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/activesupport-4.2.4/lib/active_support/notifications.rb:164:in `instrument'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/flexirest-1.2.11/lib/flexirest/request.rb:135:in `call'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/flexirest-1.2.11/lib/flexirest/mapping.rb:46:in `_call'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/flexirest-1.2.11/lib/flexirest/mapping.rb:28:in `block in _map_call'
	from (irb):15
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/railties-4.2.4/lib/rails/commands/console.rb:110:in `start'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/railties-4.2.4/lib/rails/commands/console.rb:9:in `start'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/railties-4.2.4/lib/rails/commands/commands_tasks.rb:68:in `console'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/railties-4.2.4/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/johnhenderson/.rvm/gems/ruby-2.2.3/gems/railties-4.2.4/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:8:in `require'
	from bin/rails:8:in `<main>'
```

from line 158 out of the request.rb from this block:
```
if cached
          if cached.expires && cached.expires > Time.now
            Flexirest::Logger.debug "  \033[1;4;32m#{Flexirest::NAME}\033[0m #{@instrumentation_name} - Absolutely cached copy found"
            return handle_cached_response(cached)
          elsif cached.etag.to_s != "" #present? isn't working for some reason
            Flexirest::Logger.debug "  \033[1;4;32m#{Flexirest::NAME}\033[0m #{@instrumentation_name} - Etag cached copy found with etag #{cached.etag}"
            etag = cached.etag
          end
        end
```

Is there something that I'm missing that needs to be swapped out and/or set in a config?

**Comment by andyjeffries** [comment]
Have you tried clearing your cache? If you're using Flexirest from within Rails you can use `Rails.cache.clear`.

If not, any chance you could edit request.rb locally and before line 158 put `Flexirest::Logger.debug cached` and see what goes out to the logs. (or use `puts`). It's claiming `cached` is a string rather than a `Flexirest::CachedResponse`, so I'd like to see what it thinks it is...

**Comment by sqlninja** [comment]
```Rails.cache.clear``` did it...

Thanks

**Comment by andyjeffries** [comment]
Great, glad it's all working now.

**Comment by jgujgu** [comment]
Excellent. I ran into this too.

**Comment by andyjeffries** [comment]
I've fixed this in v1.2.15 properly, so it shouldn't happen going forward.
